dts/bindings/i2c/snps,designware-i2c.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: DesignWare SPI node description
```

Comment:
Shouldn't this be I2C instead of SPI?

Reply:
indeed, copy-pasted this from my other branch and forgot to change it